formats/dsd/gencode_test.go

```diff
@@ -1,4 +1,4 @@
-//nolint:nakedret,unconvert,gocognit
+//nolint:nakedret,unconvert,gocognit,wastedassign,gofumpt
 package dsd
```

Comment:
I'm not sure disabling `wastedassign` is a good thing. Why do you think this is required here?

Reply:
This is generated code, so I see no value in changing that.
The linter complained, I ignored it.
Also, this is just for testing, performance does not really matter (so much) here.